### DISCUSSION TITLE
Is renovate checking and updating `.node-version` file?

### DISCUSSION THREAD

**Discussion by tschersach** [renovatebot/renovate#17457] [Request Help] [answered]
I'm using `fnm` and `.node-version` files for my node versioning in repos and I'm not getting updates from renovate.
Do note that we are on node 18 and I know it's not LTS but shouldn't renovate suggest 18 bumps if I'm already on 18?
I'm having a hard time finding if that question is already answered seeing that searching for `.node-version` in the repo returns a lot of unrelated things. And searching for `fnm` is not returning anything.

Regards 😄

**Comment by Gabriel-Ladzaretti** [comment, marked as answer]
You can try disabling `ignoreUnstabe` for `nodenv` 
```
"packageRules": [
  {
    "matchManagers": ["nodenv"],
    "ignoreUnstable": "false"
  }
]
```

**Reply by Gabriel-Ladzaretti** [reply]
Renovate does update `.node-version` files.

`nodenv` uses the `node` versioning which is indeed LTS aware, therefore all non LTS releases will be filtered out in the `datasource` stage regardless of which version is being used by the user.

**Reply by tschersach** [reply]
Thanks for the detailed explanation Gabriel, everything is good 😄
